Bugfix v2.0.2 (#710)
precice/precice: CHANGELOG.md

All notable changes to this project will be documented in this file. For future plans, see our [Roadmap](https://github.com/precice/precice/wiki/Roadmap).

## 2.0.2

- Fixed a critical bug in the testing framework.

> - Fixed a critical bug in the partitioning with two-level initialization enabled.

Reviewer: ```suggestion
- Fixed a critical bug in the partitioning for geometric filtering on master.
```
This also affects partitioning when two-level init is off.

Author: We should probably also mention that this is a setting in the use-mesh tag and that this bug does not affect the default.

```suggestion
- Fixed a critical bug in the partitioning for geometric filter set to `on-master` in `<use-mesh>` tags. The default configuration is `on-slave`.
```

Reviewer: I agree
